<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detailswidget</class>
 <widget class="QWidget" name="detailswidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Save Changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="discardButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Discard Changs</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>287</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="informationGB">
     <property name="title">
      <string>Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="modifiedDateLabel">
          <property name="text">
           <string>Date Modified:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dateModified">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>106</width>
            <height>16</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="modifiedByLabel">
          <property name="text">
           <string>by</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="modifiedBy">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="createdDateLabel">
          <property name="text">
           <string>Date Created:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dateEntered">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="createdByLabel">
          <property name="text">
           <string>by</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="createdBy">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QWidget" name="detailsContainer" native="true">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>400</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="descriptionBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>151</height>
         </size>
        </property>
        <property name="title">
         <string>Description:</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTextEdit" name="description"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
